<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerSetting</class>
 <widget class="QDialog" name="PlayerSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>81</width>
     <height>66</height>
    </rect>
   </property>
   <layout class="QVBoxLayout">
    <item>
     <widget class="QPushButton" name="ok_pushButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>&amp;Ok</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_pushButton">
      <property name="text">
       <string>&amp;Cancel</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_p1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>283</width>
     <height>103</height>
    </rect>
   </property>
   <property name="title">
    <string>Player 1 (Black)</string>
   </property>
   <layout class="QGridLayout" name="_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_name_p1">
      <property name="text">
       <string>&amp;Name :</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_name_p1</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_name_p1"/>
    </item>
    <item row="0" column="2" rowspan="2">
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>31</width>
        <height>41</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_type_p1">
      <property name="text">
       <string>&amp;Type :</string>
      </property>
      <property name="buddy">
       <cstring>comboBox_type_p1</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_type_p1">
      <item>
       <property name="text">
        <string>Human</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Computer ( Beginner )</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Computer ( Amateur )</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Computer (  Expert   )</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_p2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>103</y>
     <width>283</width>
     <height>103</height>
    </rect>
   </property>
   <property name="title">
    <string>Player 2 (White)</string>
   </property>
   <layout class="QGridLayout" name="_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_name_p2">
      <property name="text">
       <string>&amp;Name :</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_name_p2</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_name_p2"/>
    </item>
    <item row="0" column="2" rowspan="2">
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>51</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_type_p2">
      <property name="text">
       <string>&amp;Type :</string>
      </property>
      <property name="buddy">
       <cstring>comboBox_type_p2</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_type_p2">
      <item>
       <property name="text">
        <string>Human</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Computer ( Beginner )</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Computer ( Amateur )</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Computer ( Expert )</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
